clustering/infinispan/extension/pom.xml

```diff
@@ -90,6 +90,10 @@
             <groupId>net.jcip</groupId>
             <artifactId>jcip-annotations</artifactId>
         </dependency>
+        <dependency>
+            <artifactId>rxjava</artifactId>
+            <groupId>io.reactivex.rxjava2</groupId>
```

Comment:
This is a bit nitpicky, but these elements should be reversed to be consistent with the rest of the POM.

Reply:
Hmm.  Looks like these lines got swapped during a merge. Will fix.